### PR TITLE
feature/partial-deep

### PR DESCRIPTION
**Pull request by un4uthorized**

### CODE REVIEW

File: test/objects.test.ts [hunk 1]
```diff
@@ -95,6 +95,20 @@ describe("Objects", () => {
...
+    //    ^?
+    type test0 = Expect<Equal<res0, { a?: 1; b?: 2 }>>;
+
+    type res1 = Call<Objects.PartialDeep, { a: 1; b: { c: 2 } }>;
+    //    ^?
+    type test1 = Expect<Equal<res1, { a?: 1; b?: { c?: 2 } }>>;
+
+    type res2 = Call<Objects.PartialDeep, { a: 1; b: { c: 2; d: { e: 3 } } }>;
+    //    ^?
+    type test2 = Expect<Equal<res2, { a?: 1; b?: { c?: 2; d?: { e?: 3 } } }>>;
```
Comment (ecyrbe) [inline test/objects.test.ts:109]: can you add tests with :
- an array of objects
- an object with properties being array of abjects

**Review by ecyrbe — CHANGES_REQUESTED** [review]
Thank you, here is a litle feedback.

File: test/objects.test.ts (hunk 1 above)
Comment (un4uthorized) [reply test/objects.test.ts:109]: done :)

image

Comment (ecyrbe) [reply test/objects.test.ts:109]: Oh sorry i meant a tuple. Really sorry for confusion

Comment (un4uthorized) [reply test/objects.test.ts:109]: No problem, it's done!

**Pull request merged**